main.go

```diff
@@ -135,78 +224,76 @@ func wrapMain() error {
 
 	e := os.Environ()
 	e = append(e, fmt.Sprintf("GOOGLE_APPLICATION_CREDENTIALS=%s", keyPath))
-	runner := NewEnviron(workspace.Path, e, os.Stdout, os.Stderr)
+	wd, err := os.Getwd()
```

Comment:
I did some testing and this works as is, however we can also just remove `wd` as well and use `""` instead.

This is used to set the command runner's environment:

>         // Dir specifies the working directory of the command.
>         // If Dir is the empty string, Run runs the command in the
>         // calling process's current directory.
>         Dir string
> -- https://golang.org/pkg/os/exec/#Cmd